nerfstudio/model_components/losses.py

```diff
@@ -170,3 +170,17 @@ def nerfstudio_distortion_loss(
     loss = loss + 1 / 3.0 * torch.sum(weights**2 * (ends - starts), dim=-2)
 
     return loss
+
+
+def orientation_loss(weights, normals, viewdirs):
+    """Orientation loss from refnerf method in multinerf and nerf factory."""
```

Comment:
Can you remove "in multinerf and nerf factory" and also add TensorType. I didn't get around to making this cleaner before passing off the PR.